hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/Alert.java

```diff
@@ -63,6 +63,9 @@
     private List<Note> notes;
 
     @JsonInclude(Include.NON_EMPTY)
+    private List<LifeCycle> lifecycle;
```

Comment:
> As long as you are confident that we allow only valid status transitions I guess I am OK with its allowance. I do think we need to remove the redundant explicit fields for ack and resolved as it confuses/bloats the impl. It is not required for the PR but there should be another Jira for this work, and the priority should be high.

Yes, +1 on that. It is not done on this PR due the implications on the Actions architecture, where legacy resolved and ack fields are propagated on several places (email templating included).

So, as we have planned to review the Actions architecture on this week for the metrics+alerting  effort, this removal will be done in that task.